string modality
---
bool return_value
